website/mails/mail_presends.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
As per COSDev, "avoid redundant labeling": rename this file to `presends`.

https://cosdev.readthedocs.org/en/latest/style_guides/python.html#general-naming-guidelines